src/cloud/env/variables-build.md

```diff
@@ -207,7 +207,11 @@ stage:
 -  **Default**—_Not set_
 -  **Version**—Magento 2.1.4 and later
 
- Enables or disables the [Symfony](https://symfony.com/doc/current/console/verbosity.html) debug verbosity level for your logs. Choose the level of detail provided in the logs: `-v`, `-vv`, or `-vvv`.
+Enable or disable the [Symfony](https://symfony.com/doc/current/console/verbosity.html) debug verbosity level for `bin/magento` CLI commands executed from the deployment script.
+
+To change the verbosity of the output from _successful_ `bin/magento` CLI commands, you must also set the [MIN_LOGGING_LEVEL]({{ site.baseurl }}/cloud/env/variables-global.html#min_logging_level) to `debug`.
```

Comment:
I don't think that verbosity is only for successful commands. I think this may be confusing. Perhaps keep it simple.

```suggestion
The [MIN_LOGGING_LEVEL]({{ site.baseurl }}/cloud/env/variables-global.html#min_logging_level) must be set to `debug` for the most useful output.
```

Reply:
#7708 was update from Cloud Devs that controlling the verbosity level of command output for successful commands requires the `debug` setting for MIN_LOGGING_LEVEL.

However, I think the simplification you suggested is fine.

Reply:
> I don't think that verbosity is only for successful commands. I think this may be confusing. Perhaps keep it simple.

According to update submitted by Cloud Devs in #7708, the detail level for the command output for successful commands requires setting min_logging_level to `debug`.    Which is confusing.  I simplified it as you suggested.